pyvista/core/filters/data_set.py

```diff
@@ -5213,3 +5218,128 @@ def integrate_data(self, progress_bar=False):
         alg.SetDivideAllCellDataByVolume(False)
         _update_alg(alg, progress_bar, 'Integrating Variables')
         return _get_output(alg)
+
+    def partition(self, n_partitions, generate_global_id=False, as_composite=True):
+        """Break down input dataset into requested number of partitions.
+
+        Cells on boundaries are uniquely to each partition.
+
+        It uses a DIY-based kdtree implementation that builds balances the cell
```

Comment:
To me, DIY implies that the user has to supply the implementation

Reply:
Yea, this was a copy/pasta from VTK docs, but I think it refers to https://diatomic.github.io/diy/group___algorithms.html

I'll remove it. Reader can still lookup the VTK Docs.

```suggestion
        It uses a kdtree implementation that builds balances the cell
```